cloud/cloud.go

```diff
@@ -80,12 +83,21 @@ type (
 	}
 )
 
-var debugAPIRequests bool
+var (
+	pageSize         int64 = defaultPageSize
+	debugAPIRequests bool
+)
 
 func init() {
 	if d := os.Getenv("TMC_API_DEBUG"); d == "1" || d == "true" {
 		debugAPIRequests = true
 	}
+	if sizeStr := os.Getenv("TMC_API_PAGESIZE"); sizeStr != "" {
+		size, _ := strconv.Atoi64(sizeStr)
```

Comment:
Should we handle the error here and print a warning?

Reply:
This is just for debugging purposes.